src/Plutarch/Extra/IsData.hs

```diff
@@ -73,7 +75,7 @@ import Prelude
 newtype ProductIsData a = ProductIsData {unProductIsData :: a}
```

Comment:
Why do we need/want this? What is the point?

Reply:
For example: 

https://github.com/Liqwid-Labs/agora/blob/staging/agora/Agora/Proposal.hs#L319
https://github.com/Liqwid-Labs/agora/blob/staging/agora/Agora/Proposal.hs#L636